docs-requirements.txt

```diff
@@ -3,3 +3,4 @@
 -r test-requirements.txt
 sphinx~=1.4.9
 sphinx-argparse~=0.1.15
+sphinx_rtd_theme
```

Comment:
questionable, I don't think we use this theme any more. Talk to @Mariatta about this.